### PR TITLE
Make label(s) configurable

### PR DESCRIPTION
**Pull request by hypery2k**

### CODE REVIEW

**Comment by pedroabreu** [comment]
Would it make sense to move more stuff to the provider ? Like the nr of images in the row or if subtitles are on/off ?

**Comment by hypery2k** [comment]
seems like a good idea. Maybe it should still be possible to override the global config per directive parameter

**Comment by pedroabreu** [comment]
I'll grab this PR and make it in a way where you can set the settings from the provider or as a parameter in the element. Where the provider will take priority over it

**Comment by hypery2k** [comment]
cool, sounds great. Thanks

File: src/js/gallery.js [hunk 1]
```diff
@@ -19,18 +36,19 @@
...
+
     function controller($scope){
       ionGalleryData.setGallery($scope.ionGalleryItems);
       ionGalleryData.setRowSize(parseInt($scope.ionGalleryRow));
-      
+
       var _drawGallery = function(){
         $scope.items = ionGalleryData.buildGallery();
         $scope.responsiveGrid = ionGalleryData.getGridSize();
+        $scope.labels = ionGalleryConfig.labels;
```
Comment (pedroabreu) [inline src/js/gallery.js]: Is there a reason for this to be updated every time there's a change in the items array ? I prefer to move it outside that function, or is there a use case I'm not thinking of ?

Comment (hypery2k) [inline src/js/gallery.js]: no, you're totally right. My fault ;)

**Comment by pedroabreu** [comment]
Can you do a rebase of this so it can be merged ? Sorry if this isn't how it's supposed to be (the push to this PR) but it seemed the logical way to do it

**Pull request merged**